core/src/mill/define/Cross.scala

```diff
@@ -42,18 +41,17 @@ object Cross{
   *   ...
   * }
   */
-class Cross[T](cases: Any*)
-              (implicit ci: Cross.Factory[T],
-               ctx: mill.define.Ctx) extends mill.define.Module()(ctx) {
+class Cross[T](cases: Any*)(implicit ci: Cross.Factory[T], ctx: mill.define.Ctx)
+    extends mill.define.Module()(ctx) {
 
   override lazy val millModuleDirectChildren =
     this.millInternal.reflectNestedObjects[Module] ++
-    items.collect{case (k, v: mill.define.Module) => v}
+      items.collect { case (k, v: mill.define.Module) => v }
```

Comment:
I'd prefer expressions split over multiple lines with infix operators to be vertical aligned, rather than having the subsequent expressions be indented relative to the first